multinode-demo/leader.sh

```diff
@@ -3,5 +3,10 @@
 # if RUST_LOG is unset, default to info
 export RUST_LOG=${RUST_LOG:-solana=info}
 
-sudo sysctl -w net.core.rmem_max=26214400
+if [[ "$(uname)" = "Linux" ]]; then
+  (
+    set -x
+    sudo sysctl -w net.core.rmem_max=26214400
```

Comment:
insufficient for WSL, which is also uname Linux...  again, I just left it

Reply:
Really?!  I'm not sure how I feel about that just yet, it's surprising though!